<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBCameraAddition</class>
 <widget class="QDialog" name="DBCameraAddition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>141</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>249</width>
    <height>141</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>249</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add camera to list</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #E3EAFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cameraNameInputHint">
       <property name="text">
        <string>Camera name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="visibilityRangeInputHint">
       <property name="text">
        <string>Visibility range</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="viewingAngleInputHint">
       <property name="text">
        <string>Viewing angle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cameraNameInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="viewingAngleInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="visibilityRangeInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="additionToListConfirmButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="additionToListCancelButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
